py-doc-to-xml/docx_to_xml/util.py

```diff
@@ -42,7 +42,7 @@ def split_question(s: str) -> Tuple[str, str]:
      1)
      (1)
     """
-    return split_line(s, pattern=r"\s*\(?(\d+)[).]?")
+    return split_line(s, pattern=r"\s*\(?([0-9]+)[).]?")
```

Comment:
Can we change the `[).]?` to `[). ]` so it doesn't catch (for example) 8.4.4 question 4 example word "5:00".

Reply:
I thought that there were cases where there was no whitespace between the question number and the question.  Even so, the question number should end in a '.' or ')' so I think we just want to get rid of the second '?' in the pattern.

Reply:
Yes, that should be fine.